src/sentry/models/organization.py

```diff
@@ -251,13 +252,12 @@ def get_audit_log_data(self):
         }
 
     def get_owners(self) -> Sequence[User]:
```

Comment:
Seems like the return type needs to change, `Sequence[ApiUser]`.

In the future, this could just go directly on the `organization_service` to be honest.  I'll take a pass at it after your work lands. by pulling it into my PR.

Reply:
Replaced this with `ApiUser` and made downstream changes.